### PR TITLE
Finished update to 1.18.2

### PR DESCRIPTION
**Pull request by JamesGreen31**

### CODE REVIEW

File: src/main/java/adris/altoclef/util/helpers/ItemHelper.java [hunk 1]
```diff
@@ -12,6 +13,8 @@
 import net.minecraft.item.Items;
 import net.minecraft.tag.BlockTags;
 import net.minecraft.util.DyeColor;
+import net.minecraft.util.registry.Registry;
+import org.apache.commons.compress.compressors.lz77support.LZ77Compressor;
```
Comment (adrisj7) [inline src/main/java/adris/altoclef/util/helpers/ItemHelper.java:17]: ??

File: src/main/java/adris/altoclef/util/helpers/WorldHelper.java [hunk 2]
```diff
@@ -256,6 +258,18 @@ static boolean canReach(AltoClef mod, BlockPos pos) {
         return !mod.getBlockTracker().unreachable(pos);
     }
 
+    static boolean isOcean(RegistryEntry<Biome> b){
+        return (b.matchesKey(BiomeKeys.OCEAN)
+        ||b.matchesKey(BiomeKeys.COLD_OCEAN)
+        ||b.matchesKey(BiomeKeys.DEEP_COLD_OCEAN)
```
Comment (adrisj7) [inline src/main/java/adris/altoclef/util/helpers/WorldHelper.java:264]: space

File: src/main/java/adris/altoclef/util/helpers/ItemHelper.java [hunk 3]
```diff
@@ -284,7 +289,7 @@ public static boolean areShearsEffective(Block b) {
                         || b == Blocks.DEAD_BUSH
                         || b ==Blocks.VINE
                         || b == Blocks.TRIPWIRE
-                        || BlockTags.WOOL.contains(b)
+                        || Registry.BLOCK.getKey(b).map(e -> Registry.BLOCK.entryOf(e).streamTags().anyMatch(t -> t == BlockTags.WOOL)).orElse(false)
```
Comment (adrisj7) [inline src/main/java/adris/altoclef/util/helpers/ItemHelper.java:292]: Possibly make this into a helper function, in `ItemHelper` or something

File: src/main/java/adris/altoclef/util/helpers/ItemHelper.java (hunk 1 above)
Comment (JamesGreen31) [reply src/main/java/adris/altoclef/util/helpers/ItemHelper.java:17]: woops

File: src/main/java/adris/altoclef/util/helpers/WorldHelper.java (hunk 2 above)
Comment (JamesGreen31) [reply src/main/java/adris/altoclef/util/helpers/WorldHelper.java:264]: s p a c e

Comment (adrisj7) [reply src/main/java/adris/altoclef/util/helpers/WorldHelper.java:264]: s  p  a  c  e

**Review by adrisj7 — APPROVED** [review]

**Pull request merged**